### DISCUSSION TITLE
The Vuepress style affects the component style

### DISCUSSION THREAD

**Discussion by Xyuan-abs** [vuepress/core#596] [Q&A] [answered]
The Vuepress style affects the component style。
For example，
![image](https://user-images.githubusercontent.com/64180387/147617990-abde6c66-39aa-4431-932e-d7031a3426a0.png)

How can I solve this problem？

**Comment by Mister-Hope** [comment, marked as answer]
As a site generator, it's fully resonable for any VuePress Theme normalize html tags. So that the appearance will be same in different bowsers on different platforms.

You can overide it in style files if you like. See https://v2.vuepress.vuejs.org/reference/default-theme/styles.html#palette-file
